### ISSUE TITLE
slashes: should not report http://

### ISSUE THREAD

**Issue by azu** [textlint-rule/textlint-rule-preset-google#20]
```

@textlint-rule/textlint-rule-preset-google/google-slashes: Don't use slashes to separate alternatives.
https://developers.google.com/style/slashes#slashes-with-alternatives

/Users/azu/.ghq/github.com/almin/almin/docs/tutorial/todomvc/README.md:13:18
                         v
    12.
    13. - https://github.com/almin/almin/tree/master/examples/todomvc
    14.
                         ^

@textlint-rule/textlint-rule-preset-google/google-slashes: Don't use slashes to separate alternatives.
https://developers.google.com/style/slashes#slashes-with-alternatives

/Users/azu/.ghq/github.com/almin/almin/docs/tutorial/todomvc/README.md:13:28
                                   v
    12.
    13. - https://github.com/almin/almin/tree/master/examples/todomvc
    14.
                                   ^

@textlint-rule/textlint-rule-preset-google/google-slashes: Don't use slashes to separate alternatives.
https://developers.google.com/style/slashes#slashes-with-alternatives

/Users/azu/.ghq/github.com/almin/almin/docs/tutorial/todomvc/README.md:13:39
                                              v
    12.
    13. - https://github.com/almin/almin/tree/master/examples/todomvc
    14.
                                              ^


@textlint-rule/textlint-rule-preset-google/google-slashes: Don't use slashes to separate alternatives.
https://developers.google.com/style/slashes#slashes-with-alternatives

/Users/azu/.ghq/github.com/almin/almin/docs/tutorial/counter/README.md:201:19
                           v
    200.
    201. - [almin/examples/counter/test at master · almin/almin](https://github.com/almin/almin/tree/master/examples/counter/test "almin/examples/counter/test at master · almin/almin")
    202.
                           ^
```

This is a false-positive.
